css/styles.css

```diff
@@ -44,8 +63,49 @@ a {
   color: #fff;
 }
 
-li {
-  list-style: none;
+.menu_mobile li a {
+  font-family: 'Roboto', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 24px;
+  line-height: 32px;
+  color: #edecff;
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+}
+
+.menu_mobile {
```

Comment:
- [ ] Great job @NeckerFree! However, the hamburger button and mobile menu still need to be changed somehow to meet the design guidelines from [the chosen Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). Please take a look at the below screenshots 👇 
- In your style close button is placed outside the menu 👇 
![Screenshot (227)](https://user-images.githubusercontent.com/67087664/157299705-6b348094-4540-40b4-a687-3a7d1535e9cd.png)
![Screenshot (228)](https://user-images.githubusercontent.com/67087664/157299716-700722bc-2473-41cc-9fc4-1d6564aa87a7.png)
- Figma template 👇 
![Screenshot (229)](https://user-images.githubusercontent.com/67087664/157299912-ce0bef5e-b1fb-466c-b055-bbea06c25917.png)

Reply:
Issue Fixed:
![Menu_T](https://user-images.githubusercontent.com/8497300/157371236-66c5c886-3665-4776-a874-0cbbfa6748a8.png)